### PR TITLE
[ALLUXIO-1722] Translate the page Configuring-Alluxio-with-secure-HDFS.md into Chinese

### PR DESCRIPTION
**Pull request by saltylin**
https://tachyon.atlassian.net/browse/ALLUXIO-1722

### CODE REVIEW

File: docs/cn/Configuring-Alluxio-with-secure-HDFS.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+layout: global
+title: 在HDFS安全认证模式上配置Alluxio
+nickname: Alluxio使用secure HDFS
+group: Under Store
+priority: 3
+---
+
+该向导介绍如何配置Alluxio从而使用安全认证模式下的[HDFS](https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-hdfs/HdfsUserGuide.html)作为底层文件系统。Alluxio支持安全认证模式下的HDFS作为底层文件系统，通过[Kerberos]认证。
+
```
Comment (ifcharming) [inline docs/cn/Configuring-Alluxio-with-secure-HDFS.md]: [Kerberos](http://web.mit.edu/kerberos/)

**Comment by ifcharming** [comment]
Thank you for the translation! Left a minor comment.

File: docs/cn/Configuring-Alluxio-with-secure-HDFS.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+---
+layout: global
+title: 在HDFS安全认证模式上配置Alluxio
+nickname: Alluxio使用secure HDFS
+group: Under Store
+priority: 3
+---
+
+该向导介绍如何配置Alluxio从而使用安全认证模式下的[HDFS](https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-hdfs/HdfsUserGuide.html)作为底层文件系统。Alluxio支持安全认证模式下的HDFS作为底层文件系统，通过[Kerberos]认证。
```
Comment (RongGu) [inline docs/cn/Configuring-Alluxio-with-secure-HDFS.md]: "向导"->"指南"

File: docs/cn/Configuring-Alluxio-with-secure-HDFS.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+priority: 3
+---
+
+该向导介绍如何配置Alluxio从而使用安全认证模式下的[HDFS](https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-hdfs/HdfsUserGuide.html)作为底层文件系统。Alluxio支持安全认证模式下的HDFS作为底层文件系统，通过[Kerberos]认证。
+
+注：在HDFS安全认证模式下的Kerberos认证不是Alluxio通过Kerberos进行内部的认证
+
+# 初始步骤
+
+要在许多机器上运行Alluxio集群，需要在这些机器上部署二进制文件。你可以自己[编译Alluxio](Building-Alluxio-Master-Branch.html)，或者[下载二进制包](Running-Alluxio-Locally.html)
```
Comment (RongGu) [inline docs/cn/Configuring-Alluxio-with-secure-HDFS.md]: “二进制文件”->“二进制包”

**Comment by RongGu** [comment]
Thanks, left a few comments:)

**Comment by apc999** [comment]
+1 LGTM, thanks for the patch!

**Pull request merged**